WordPress/Classes/ViewRelated/Themes/ThemeBrowser.storyboard

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I've noticed this storyboard contains a single ViewController. Is that expected to grow?

(Otherwise we may just wanna use a single XIB, and the initialization code would shrink to just two lines!)